src/templates/pages/styleguide/forms.twig

```diff
@@ -245,6 +245,28 @@
                 {% endblock %}
             {% endembed %}
 
+            {% embed 'styleguide/block' %}
+                {% block heading %}Messages{% endblock %}
+                {% block content %}
```

Comment:
can you add a `{% block markup %}` with the same html and then the markup will be visible in the browser.